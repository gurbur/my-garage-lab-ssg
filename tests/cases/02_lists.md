- 순서 없는 리스트 1번 항목
- 2번 항목
    - 중첩된 1번 항목
        - 더 깊게 중첩된 항목
    - 중첩된 2번 항목
- 3번 항목

1. 순서있는 리스트 1번
2. 2번 항목
    1. 중첩된 순서 있는 리스트
    2. 항목 2
        - 중첩된 순서 없는 리스트
3. 3번 항목
