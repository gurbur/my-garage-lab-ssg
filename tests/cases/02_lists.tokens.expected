[DASH]
[TEXT]{" 순서 없는 리스트 "}
[NUMBER]{"1"}
[TEXT]{"번 항목"}
[NEWLINE]
[DASH]
[TEXT]{" "}
[NUMBER]{"2"}
[TEXT]{"번 항목"}
[NEWLINE]
[TAB]
[DASH]
[TEXT]{" 중첩된 "}
[NUMBER]{"1"}
[TEXT]{"번 항목"}
[NEWLINE]
[TAB]
[TAB]
[DASH]
[TEXT]{" 더 깊게 중첩된 항목"}
[NEWLINE]
[TAB]
[TAB]
[DASH]
[TEXT]{" 중첩된 "}
[NUMBER]{"2"}
[TEXT]{"번 항목"}
[NEWLINE]
[DASH]
[TEXT]{" "}
[NUMBER]{"3"}
[TEXT]{"번 항목"}
[NEWLINE]
[NEWLINE]
[NUMBER]{"1"}
[DOT]
[TEXT]{" 순서있는 리스트 "}
[NUMBER]{"1"}
[TEXT]{"번"}
[NEWLINE]
[NUMBER]{"2"}
[DOT]
[TEXT]{" "}
[NUMBER]{"2"}
[TEXT]{"번 항목"}
[NEWLINE]
[TAB]
[NUMBER]{"1"}
[DOT]
[TEXT]{" 중첩된 순서 있는 리스트"}
[NEWLINE]
[TAB]
[NUMBER]{"2"}
[DOT]
[TEXT]{" 항목 "}
[NUMBER]{"2"}
[NEWLINE]
[TAB]
[TAB]
[DASH]
[TEXT]{" 중첩된 순서 없는 리스트"}
[NEWLINE]
[NUMBER]{"3"}
[DOT]
[TEXT]{" "}
[NUMBER]{"3"}
[TEXT]{"번 항목"}
[NEWLINE]
[EOF]
